sources/wakatime_formatter.py

```
@@ -0,0 +1,179 @@
+"""
+WakaTime Formatter - Formats WakaTime statistics for README display.
+
+This module provides formatting functions to convert WakaTime API data
+into markdown-formatted text with progress bars for GitHub profile READMEs.
+"""
+
+from typing import Dict, List, Optional
```

Ruff (0.14.10)

`typing.Dict` is deprecated, use `dict` instead

(UP035)